session/session.go

```diff
@@ -123,7 +123,7 @@ type Session interface {
 	LastMessage() string  // LastMessage is the info message that may be generated by last command
 	AffectedRows() uint64 // Affected rows by latest executed stmt.
 	// Execute is deprecated, use ExecuteStmt() instead.
-	Execute(context.Context, string) ([]sqlexec.RecordSet, error) // Execute a sql statement.
+	Execute(context.Context, string) (sqlexec.RecordSet, error) // Execute a sql statement.
```

Comment:
Maybe we can remove this method totally?
It has already been marked as deprecated ... we can use the `ExecuteInternal` API which is safer.
And for test code, we can for example provide a function for the `testkit` package.

```
Execute(ctx context.Context, se session.Session, sql string)  (sqlexec.RecordSet, error)
```

Reply:
👍 removal will make it easier to fix the plugins.